index.js

```diff
@@ -685,7 +685,14 @@ Client.prototype.publishersInfo = function (publishers, callback) {
   if (!Array.isArray(publishers)) publishers = [ publishers ]
   if (publishers.length === 0) return
 
-  this.publishers = this.publishers ? this.publishers.concat(publishers) : underscore.clone(publishers)
+  if (this.publishers && this.publishers.length) {
```

Comment:
maybe `this.publishers.length > 0`to maintains the style of the length check done above?

isn't there a race in this check that could result in two calls to `_publishersInfo` in flight? since `_publishersInfo` still calls itself once more (via setTimeout) when `underscore.rest()` returns an empty array

Reply:
well, the only solid requirement is that things pass `standard` ... when other folks edit modules i've written, as long as it passes `standard`, i don't put up a fuss as to whether or not the edits are consistent with my style. nonetheless, it's good for me to be consistent.

you're right in that there may be some overlapping. that is probably inevitable in the sense that we don't want to wait until one request returns before scheduling the next one...